package.json

```diff
@@ -50,6 +49,7 @@
     "browser-sync-webpack-plugin": "^2.0.1",
     "chai": "^4.2.0",
     "css-loader": "^5.1.0",
+    "cypress": "^6.6.0",
```

Comment:
Is there a reason why you moved this here?

Reply:
It seems the cypress that I downloaded is the version 6.6.0

Reply:
Can you revert this change then (including the version back to 6.5.0)? this change is moving cypress out of dependencies

Reply:
done